.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,25 +1,57 @@
 ---
 name: Bug Report
 about: Something doesn't work correctly in Atmosphère.
-
+#assignees:
 ---
 
 ## Bug Report
 
+[ If any section does not apply, use N/A. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
+
+[ * ]  
+[ Note: If the bug or crash you encountered is related to; ]  
+[ - software used to make "backups", ]  
+[ - software explicitly distributed for piracy, ]  
+[ - other software related to the duplication of copyrighted works,  
+no support for your issue will be provided and your issue will be closed. ]  
```

Comment:
Sorry about the nitpick, but the way this line is wrapped is inconsistent with the rest of the document. Everywhere else, lines aren't wrapped. If you think this line should be wrapped, should it have an individual set of brackets for each line?

Reply:
I fiddled around with it and decided I had no idea what I wanted. I'll fully wrap both lines just to be careful.